Sources/ForageSDK/Foundation/Telemetry/ForageLogger.swift

```diff
@@ -78,17 +78,24 @@ internal protocol ForageLogger {
     /// - Parameter error: An optional error object associated with the critical log.
     /// - Parameter attributes: An optional dictionary of key-value pairs providing additional context for the log.
     func critical(_ message: String, error: Error?, attributes: [String: Encodable]?)
+    
+    /// The ForageSDK object needs a handler to the TraceId in order to pass it along to each endpoint call.
+    /// - Returns: The traceID that was generated at the beginning of the app's lifecycle.
+    func getTraceId() -> String
 }
 
 /// Read this [document](https://docs.google.com/document/d/1BU609qv7qSDN-tdNaJP-sAyrgeM6REe9tvcMk5CxN3c/edit#bookmark=id.w44b8kk07o7o) for to learn more about how DatadogLogger works
 internal class DatadogLogger : ForageLogger {
     private static let DD_CLIENT_TOKEN: String = "pub1e4572ba0f5e53df108c333d5ec66c02"
     private static let DD_SERVICE_NAME: String = "ios-sdk"
     private static let DD_SDK_INSTANCE_NAME: String = "forage"
+    
+    // DO NOT UPDATE! Generate 1 TraceID per living session of the app
+    internal static var traceId: String = generateTraceId()
```

Comment:
```suggestion
    // DO NOT UPDATE! Generate 1 TraceID per living session of the app
    internal static let traceId: String = generateTraceId()
```

we can keep the comment, but this can programmatically enforce this